src/types.rs

```diff
@@ -72,11 +75,6 @@ impl Id {
         let array = self.0.to_be_bytes();
 
         for i in 0..array.len() {
-            if array[i] == 0 && i != (array.len() - 1) {
```

Comment:
If leading zeros are not skipped anymore this comment should probably be removed

Reply:
good point, fixed

Reply:
I fear this will break backwards compatibility, no?